### PR TITLE
Build UI against Nessie API v1 from 0.44.0

### PR DESCRIPTION
**Pull request by dimas-b**
This is a temporary pin in preparation for API V2.

Newer builds will not publish OpenAPI YAML, which is why it is
necessary to refer to an older Nessie version in the UI build script.

The build uses a checked-in copy of Nessie 0.44.0 OpenAPI YAML file
at ui/src/openapi/

UI is to be updated to the upcoming V2 Nessie API eventually.

### CODE REVIEW

**Review by XN137 — APPROVED** [review]
lgtm since it is a temporary thing and i assume you tried out the UI locally.

do we have a rough timeline when we could be updating the UI to v2 api?
we should file a ticket to not forget about it

**Comment by dimas-b** [comment]
I've reworked this PR to use a check-in copy of Nessie API v1 from 0.44.0 to avoid ivy repository workarounds at build time.

The API v1 YAML is not expected to evolve.

**Comment by dimas-b** [comment]
And, yes, I did manually verify that the UI still works :)

File: ui/src/openapi/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,2324 @@
+---
+# A copy of OpenAPI definition of Nessie 0.44.0, a.k.a. API v1
```
Comment (XN137) [inline ui/src/openapi/openapi.yaml:2]: shouldnt the filename reflect this i.e. `openapi-nessie044.yaml` or the like?
imo then its clearer this is a temporary thing potentially, not the "leading" openapi file

**Review by XN137 — DISMISSED** [review]

File: ui/src/openapi/openapi.yaml (hunk 1 above)
Comment (dimas-b) [reply ui/src/openapi/openapi.yaml]: renamed

**Review by XN137 — APPROVED** [review]

**Pull request merged**